test/suites/storage_driver_dir.sh

```diff
@@ -1,11 +1,10 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Do we need this line at all?

Reply:
The `.shellcheckrc` file made those shebang changes unnecessary.